mvc/src/test/java/nextstep/mvc/controller/tobe/AnnotationHandlerMappingTest.java

```diff
@@ -30,7 +29,7 @@ void get() throws Exception {
         when(request.getRequestURI()).thenReturn("/get-test");
         when(request.getMethod()).thenReturn("GET");
 
-        final var handlerExecution = (HandlerExecution) handlerMapping.getHandler(request);
+        final var handlerExecution = (HandlerExecution)handlerMapping.getHandler(request);
         final var modelAndView = handlerExecution.handle(request, response);
```

Comment:
```suggestion
        final var handlerExecution = (HandlerExecution) handlerMapping.getHandler(request);
        final var modelAndView = handlerExecution.handle(request, response);
```
형변환시 공백을 빼주셨는데 이유가 있으신가요? (컨밴션시에 공백이 있는 걸로 알고있어서 질문드립니다!)
그리고 어떨때 var, 어떨때 타입을 명시적으로 사용해주는데 어떤 차이때 var 사용하시는지 궁금합니다! 
(레거시 코드는 var가 많은건 알고 있어서 새로 작성해주신 코드에서만 말씀해주시면 돼요!)

Reply:
공백의 경우 네이버 코딩 컨벤션에 맞춰서 자동 수정이 되다보니 공백이 빠진 상태로 진행된 것 같습니다..! (현재 팀 컨벤션이 네이버 코딩 컨벤션이라 이에 맞춰 진행하였습니다) 
var 같은 경우는 처음에 테스트 짜면서 var로 해놓은 부분을 다 수정하지 않았던 것 같네요. 일괄적으로 바꿔두겠습니다.